allauth/account/tests/test_signup.py

```diff
@@ -315,3 +315,17 @@ def test_django_password_validation(self):
                 "password1",
                 ["This password is too short. It must contain at least 9 characters."],
             )
+
+    def test_get_initial_with_valid_email(self):
```

Comment:
The codebase is transitioning towards using `pytest` based tests. Can you turn this into a regular (pytest) function? As in, not make it a mapper of a class, let it accept `rf` (request factory) as a parameter, and so on...